packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts

```diff
@@ -118,4 +110,16 @@ export class BlockchainPropertiesService {
 
     private assure0x = (privateKey: string) =>
         privateKey.startsWith('0x') ? privateKey : `0x${privateKey}`;
+
+    private async waitForPropertiesToBeDeployed() {
+        try {
+            await waitForState(
+                async () => await this.isDeployed(),
+                'Blockchain properties were not deployed',
+                { interval: 5_000, maxTries: 24 }
+            );
+        } catch (e) {
+            throw Error(e);
```

Comment:
What the hell? Why it's not `new Error` and why it just rethrows? Is `waitForState` throwing just string (e.g. `throw "some error message"`) not wrapped with error object?

Reply:
> Why it's not new Error 
my bad
>  Is waitForState throwing just string 
yes

Reply:
So cannot waitForState throw normal error?

Reply:
Done